Consul.Test/OperatorTest.cs

```diff
@@ -145,6 +145,70 @@ public async Task Operator_AutopilotGetHealth_ReturnsHealthStatus()
             }
         }
 
+        [Fact]
+        public async Task Operator_AutopilotGetState_ReturnsStateInformation()
+        {
+            var result = await _client.Operator.AutopilotGetState();
+            Assert.NotNull(result);
+            Assert.NotNull(result.Response);
+            Assert.NotNull(result.Response.Servers);
+
+            var state = result.Response;
+            Assert.IsType<Dictionary<string, AutopilotServerState>>(state.Servers);
+            Assert.True(state.FailureTolerance >= 0);
+            Assert.True(state.OptimisticFailureTolerance >= 0);
+            Assert.NotNull(state.Voters);
+
+            if (state.Servers.Count > 0)
```

Comment:
The test code shouldn't have if conditions, which may lead to not executing a whole section of test code.